pandas/tools/util.py

```diff
@@ -74,9 +94,33 @@ def to_numeric(arg, errors='raise'):
     >>> import pandas as pd
     >>> s = pd.Series(['1.0', '2', -3])
     >>> pd.to_numeric(s)
+    0    1.0
+    1    2.0
+    2   -3.0
+    dtype: float64
+    >>> pd.to_numeric(s, downcast='float')
+    0    1.0
+    1    2.0
+    2   -3.0
+    dtype: float16
```

Comment:
It depends on what your platform supports.  Look at `np.typecodes['Float'][0]` and see what it is for your own machine.